vmm/src/cpu.rs

```diff
@@ -379,14 +376,31 @@ impl Vcpu {
         })))
     }
 
+    #[cfg(target_arch = "aarch64")]
+    /// Configures a aarch64 specific vcpu and should be called once per vcpu when created.
+    ///
+    /// # Arguments
+    ///
+    /// * `machine_config` - Specifies necessary info used for the CPUID configuration.
+    /// * `kernel_entry_point` - Kernel entry point address in guest memory and boot protocol used.
+    /// * `vm_memory` - Guest memory.
+    pub fn configure(
+        &mut self,
+        #[allow(unused_variables)] kernel_entry_point: Option<EntryPoint>,
+        #[allow(unused_variables)] vm_memory: &GuestMemoryAtomic<GuestMemoryMmap>,
+    ) -> Result<()> {
+        unimplemented!();
+    }
+
     #[cfg(target_arch = "x86_64")]
-    /// Configures a x86_64 specific vcpu and should be called once per vcpu from the vcpu's thread.
+    /// Configures a x86_64 specific vcpu and should be called once per vcpu when created.
     ///
     /// # Arguments
     ///
     /// * `machine_config` - Specifies necessary info used for the CPUID configuration.
     /// * `kernel_entry_point` - Kernel entry point address in guest memory and boot protocol used.
-    /// * `vm` - The virtual machine this vcpu will get attached to.
+    /// * `vm_memory` - Guest memory.
+    /// * `cpuid` - CpuId, wrapper over the `kvm_cpuid2` structure.
     pub fn configure(
```

Comment:
Could this be in the arch crate?

Reply:
It is possible.
I am trying this, will submit another commit to move it.